libraries/botbuilder-azure-blobs/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "botbuilder-azure-blobs",
+  "author": "Microsoft Corp.",
+  "description": "Botbuild storage bindings forto Azure Blobs services",
+  "version": "4.1.6",
+  "license": "MIT",
+  "keywords": [
+    "botbuilder",
+    "botframework",
+    "bots",
+    "chatbots"
+  ],
+  "bugs": {
+    "url": "https://github.com/Microsoft/botbuilder-js/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Microsoft/botbuilder-js.git"
+  },
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "typesVersions": {
```

Comment:
Note: this will be our pattern for upgrading Typescript while maintaining compatibility with older versions.